src/decayangle/decay_topology.py

```diff
@@ -696,6 +696,7 @@ def boost(
         inverse: bool = False,
         tol: Optional[float] = None,
         convention: Literal["helicity", "minus_phi", "canonical"] = "helicity",
+        last_boost: bool = True,
```

Comment:
fills bad esthetically: you cover a small corner case (massless) with a flag in a main general function.
I'd drop the flag and stick with true value.

Reply:
I did not get before what this flag is doing, when writing the previous message was thinking something else.

Can one check if the mass, and Boost to other frame, e.g. say `gamma=5` if mass is zero.

Reply:
I think this is not nescessary...
Just leaving the last boost is enough, since we have
$ (R_z^j R_y^j B_z^{j1} R_z^j R_y^j R_z^j B_z^{j2})^{-1} \times R_z R_y B_z^1 R_z R_y R_z B_z^2$
This is equal to
$ \times R_z R_y B_z^1 R_z R_y R_z B_z^2$

Reply:
Would you be okay, with just never doing the final boost?
The tests show, that is produces equivalent behavior.